search/src/main/scala/weco/api/search/elasticsearch/ImagesQuery.scala

```diff
@@ -40,56 +43,42 @@ case object ImageSimilarity {
 }
 
 case object ImagesMultiMatcher {
-
-  private def boostedWorkFields(
-    boost: Int,
-    fields: Seq[String]
-  ): Seq[FieldWithOptionalBoost] =
-    fields.map(f => FieldWithBoost(s"source.$f", boost))
-
-  private def workFields(fields: Seq[String]): Seq[FieldWithOptionalBoost] =
-    fields.map(f => FieldWithoutBoost(s"source.$f"))
-
   private val titleFields = Seq(
-    "data.alternativeTitles",
-    "data.title.english",
-    "data.title.shingles",
-    "data.title"
+    "query.source.alternativeTitles",
+    "query.source.title.english",
+    "query.source.title.shingles",
+    "query.source.title"
   )
 
-  private val idFields: Seq[FieldWithOptionalBoost] = workFields(
-    fields = Seq(
-      "id.canonicalId",
-      "id.sourceIdentifier.value",
-      "data.otherIdentifiers.value"
-    )
-  ) ++ Seq(
-    "state.canonicalId",
-    "state.sourceIdentifier.value"
-  ).map(FieldWithoutBoost(_))
+  private val idFields: Seq[FieldWithOptionalBoost] =
+    Seq(
```

Comment:
There are only four fields here, where there were five.  I assume that's because id.canonicalId and state.canonicalId are duplicates?

Reply:
No, it’s `id.sourceIdentifier.value` and `data.otherIdentifiers.value`, which are two work fields that collapsed into `query.identifiers` – although we distinguish between source/other identifiers in the pipeline, whenever we search we want both, so I collapsed them into a single list.

```
source.id.canonicalId              ~> query.source.id
source.id.sourceIdentifier.value   ~> query.source.identifiers.value
source.data.otherIdentifiers.value ~> query.source.identifiers.value
state.canonicalId                  ~> query.id
state.sourceIdentifier.value       ~> query.sourceIdentifier.value
```